### ISSUE TITLE
+40 Users

### ISSUE THREAD

**Issue by mikithd** [hl5ky/dvsmu#1]
hi, since I have a system with more than 8 GB of RAM con CPU di 3 GHz, is it possible to manage more than 40 users?  Thanks and congratulations for the work!!!

**Comment by hl5ky** [comment]
I just wrote the software only in Korean and mostly for Koreans. If you are not a Korean and still use it, congratulations.

Having more than 40 users is up to not only the hardware capability also the number of active users. I can't exactly tell you can do that or not. But because most of time users do not operate at the same time, you can probably manage more than 40 users with such a good system.

I am sorry but I don't have a plan to do that myself. I've almost forgot what I did for the dvsmu. lol

**Comment by mikithd** [comment]
You are absolutely right, in this case what matters is not only the performance that the hardware system can have but also the quality of the internet connection and, I can say with certainty, that I have no problems on all aspects.
First of all I want to congratulate you once again for the work you have done and continue to do to make us radio amateurs "entertain". I take this opportunity to tell you that, by making small changes to the dvsmu file in /usr/local/dvs, I was able to add 5 more users. The latter additions are fully functional with all the related services combined.

Thank you again.
